src/main/scala/org/constellation/DAO.scala

```diff
@@ -263,7 +265,17 @@ class DAO(
         Blocker.liftExecutionContext(unboundedHealthExecutionContext)
       )(ce, timer, cs)
     }
-    peerHealthCheckWatcher = PeerHealthCheckWatcher(ConfigUtil.config, peerHealthCheck, unboundedHealthExecutionContext)
+
+    healthCheckConsensusManager = new HealthCheckConsensusManager(
```

Comment:
please use apply fn for instantiating a new class, as we do with most of the services

Reply:
Sure, I will.